### ISSUE TITLE
[Screen readers - API Management - Table]: Screen Reader is not identifying the visually appearing table present, while navigating using table shortcut key 'T'.

### ISSUE THREAD

**Issue by JMach1** [paperbits/paperbits-demo#216]
### TL;DR

Table widget is not recognized as a table by Narrator.

component: paperbits-core\src\table

Related issue (same component): https://github.com/paperbits/paperbits-demo/issues/215

### MS task

https://msazure.visualstudio.com/One/_workitems/edit/15430867

User Experience:
Users Who rely on screen readers are getting confused and will not know that table is present in the page if narrator is not recognizing the table present in the page while navigating using table shortcut key 'T' hence get to failed proper information.

Note: User credentials should NOT be included in the bug.
Repro-Steps:
Open Azure portal with valid credentials and create a API Management.
Select Developer Portal (Preview) button and select it.
Navigate to the [https://accessibility.developer.azure-api.net/signin-sso?token=1&201906090559&Sg/pisCBkhUDO4/kkfFGp5bBz7eQmw3UXhNmBz+YTvVnNuwUXn1KIx657NMEkAW86ywLo2OiBTGd3NTNbLR3Qw== ](https://accessibility.developer.azure-api.net/signin-sso?token=1&201906090559&Sg/pisCBkhUDO4/kkfFGp5bBz7eQmw3UXhNmBz+YTvVnNuwUXn1KIx657NMEkAW86ywLo2OiBTGd3NTNbLR3Qw==) 
Navigate to the Left nav and select Pages button.
Navigate to the Pages window. select API Reference option.
Navigate to the any editable control details and select Add widget.
Navigate to the Table and select it.
Table will be added.
Verify whether screen Reader is identifying the visually appearing table present, while navigating using table shortcut key 'T'.
Actual Result:
Screen Reader is not identifying the visually appearing table present in the page while navigating using table shortcut key 'T'.

Observation:

When user invokes Table shortcut key(T), narrator is narrating as "There are no table in the page".

JAWS: When user invokes Table shortcut key(T), JAWS is narrating as "There are no table in the page".

NVDA: When user invokes Table shortcut key(T), Narrator is narrating as "There are no table in the page".

Expected Result:
Screen reader should be identify the visually appearing table present in the page while navigating using table shortcut key 'T'.

Ex: When user invokes Table shortcut key(T), narrator should narrate as "Cell Table with 1 row and 3 Columns".

Note:
Screen Reader is not identifying the visually appearing table present, while navigating using table shortcut key 'T'.throughout the Application.
MAS Reference:
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)
Reference Links:
[Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-poyad%40microsoft.com%7cc11478591b5a474f6d9d08d7199726c0%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637006009300301864&sdata=ZTWYamT1yE7yko5QU35ZJLf/5ZydtDeGAKrQGUiugJs%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDhmOTU2NWU5ZjQwMDRhN2Y4ZTVkNDEyNjI5YjdkMmRhfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3OTg5Mzc0OTg3NDkwOTF8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT0%3D&sdata=WFk4dHdKak5LMHVueDVLdUxzUm16cDQzaDJHcElHL09hSk1TZE4xS0E5Yz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-arhamimam%40microsoft.com) “HCL Staff should not log any third-party external bugs. Should be routed to eDAD team”.
Please reach out to [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0) for any process related queries.
